apps/omg_watcher/mix.exs

```diff
@@ -48,7 +55,8 @@ defmodule OMG.Watcher.Mixfile do
       # here only to leverage common test helpers and code
       {:fake_server, "~> 1.5", only: [:dev, :test], runtime: false},
       {:briefly, "~> 0.3.0", only: [:dev, :test], runtime: false},
-      {:omg_child_chain, in_umbrella: true, only: [:test], runtime: false}
+      {:omg_child_chain, in_umbrella: true, only: [:test], runtime: false},
+      {:phoenix, "~> 1.3", runtime: false}
```

Comment:
Just curious for me to learn, why does watcher have a phoenix dependency? It's not a webserver, or is it?

Reply:
Ah, nvm...for testing!

Reply:
yep!